db/migrate/20200419132253_devise_create_doctors.rb

```diff
@@ -3,6 +3,10 @@
 class DeviseCreateDoctors < ActiveRecord::Migration[5.2]
   def change
     create_table :doctors do |t|
+      t.string :doctorname
```

Comment:
`name` でも良いかなと思いました。
もしdoctorというprefixを付けたい場合は、 `doctor_name` にしましょう。
```suggestion
      t.string :name
```